### ISSUE TITLE
CreateContract is deprecated

### ISSUE THREAD

**Issue by jankallewulf** [OXIDprojects/arvato-afterpay-module#3]
The CreateContract method is deprecated. 

In arvato-afterpay-module/Application/Model/PaymentGateway.php following code snippet can be changed in order to avoid that the CreateContract method is executed:

function **handleDebitNote**

```
// Create Contract
$afterpayCheckoutId = Registry::getSession()->getVariable('arvatoAfterpayCheckoutId');
$service = $this->getCreateContractService($afterpayCheckoutId);
$paymentType = $oOrder->oxorder__oxpaymenttype->value;
return $service->createContract($paymentType, $apdebitbankaccount, $apdebitbankcode);
```

Replace to

`return true;`

**Comment by mariolorenz** [comment]
set CreateContract as depreceated

**Comment by mariolorenz** [comment]
fixed in new Version v2.0.5
